### ISSUE TITLE
Split is undefined

### ISSUE THREAD

**Issue by badams2527** [neumino/rethinkdbdash#70]
I tried doing a simple split on a string, and it returned that the function was undefined. Something like the following:

```javascript
r.table('companies').filter(r.row('name').downcase().split('test').count().ge(2)).run().then(function (result) {
    callback(result);
});
```

It returned that the function split was undefined. I got around it for now by doing a filter in pure JavaScript since it's a small dataset, but it may be affecting others.

**Comment by neumino** [comment]
Arg, thanks, it looks like I just forgot it.
Fix coming.

**Comment by badams2527** [comment]
Great, thank you!

**Comment by neumino** [comment]
Thanks @badams2527 for the report. It's fixed in 1.16.12.
